model.py

```diff
@@ -149,7 +149,7 @@ def toggle(self, uid, user_profile):
         return u'Your name was {} {}!'.format(action, self.title)
 
     def render_text(self):
-        title = util.make_html_bold(self.title)
+        title = util.make_html_bold(self.title) + "|  Number of Votes: " + str(len(people))
```

Comment:
This might be cleaner: Option title (5 👥)
It might also be cleaner to hide it when the count is zero.